lib/collection/benches/hash_ring_bench.rs

```diff
@@ -0,0 +1,40 @@
+use collection::hash_ring::HashRing;
+use criterion::{criterion_group, criterion_main, Criterion};
+use rand::Rng;
+
+fn hash_ring_bench(c: &mut Criterion) {
```

Comment:
Out of curiosity, what was the overhead of the fair ring over the raw ring?

Reply:
30 nanoseconds